src/console/components/console.js

```diff
@@ -55,25 +71,25 @@ export default class ConsoleComponent {
       break;
     case KeyboardEvent.DOM_VK_M:
       if (e.ctrlKey) {
-        e.stopPropagation();
-        e.preventDefault();
-        return this.onEntered(e.target.value);
+        this.doEnter(e);
```

Comment:
Does this need a `return` as well?

Reply:
nice catch! thx!

Reply:
oh, I missed your fixed comment :-p
Original code have `return`.
I think we should confirm to @ueokande .